library.properties

```diff
@@ -1,5 +1,5 @@
 name=Ethernet
-version=2.0.2
+version=2.0.3
```

Comment:
Why increase the version number of the library? I would expect this is up to the maintainers of the repository. Should be a separate PR in my opinion.

Reply:
@gudnimg is right. The release manager takes care of bumping the version in this field at the time of preparation for a new release. Bumping it before that time could cause confusion.

Reply:
Okay, sorry. Reverted this change.